provider/runbook_resource_test.go

```diff
@@ -303,6 +317,9 @@ resource "firehydrant_runbook" "test_runbook" {
 
 func testAccRunbookResourceConfig_update(rName string) string {
 	return fmt.Sprintf(`
+resource "firehydrant_team" "test_team1" {
+	name = "test-team1-%s"
+}
```

Comment:
same as above, terraform fmt
```suggestion
resource "firehydrant_team" "test_team1" {
  name = "test-team1-%s"
}
```